packages/snaps-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.2.0]
+
+### Added
+
+- Debounce persistence of state updates ([#3258](https://github.com/MetaMask/snaps/pull/3258))
```

Comment:
Should this be under "Changed"?

Reply:
🤷 depends if we think of it as a feature 😄